src/mmwtex.c

```diff
@@ -853,10 +853,10 @@ long texDefWhiteSpaceLen(char *ptr)
     i++;
   }
   bug(2307);
-  return 0; // Dummy return - never executed
+  assert(0);
```

Comment:
This is incorrect, `bug` can return! The user is provided a prompt and one of the options is to continue.

Reply:
So, I restore `return 0;` but remove its comment ?

Reply:
done in 811908c3ef9857018473e9d66022cda45bc0e6dd